docs/runtime/shell.md

```diff
@@ -99,6 +99,7 @@ console.log(stderr);
 The default handling of non-zero exit codes can be configured by calling `.nothrow()` or `.throws(boolean)` on the `$` function itself.
 
 ```js
+import { $ } from "bun";
```

Comment:
I think the import is essential (like in all the other code snippets) to clarify the `throws`/`nothrow` applied to the imported function.